electron-react/src/pages/ImportHexKey.js

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import { makeStyles } from "@material-ui/core/styles";
+import Container from "@material-ui/core/Container";
+import logo from "../assets/img/chia_logo.svg"; // Tell webpack this JS file uses this image
+import { withRouter } from "react-router-dom";
+import { connect, useDispatch } from "react-redux";
+import Link from "@material-ui/core/Link";
+import ArrowBackIosIcon from "@material-ui/icons/ArrowBackIos";
+import Button from "@material-ui/core/Button";
+import CssTextField from "../components/cssTextField";
+import { changeEntranceMenu, presentSelectKeys } from "../modules/entranceMenu";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    background: "linear-gradient(45deg, #181818 30%, #333333 90%)",
+    height: "100%"
+  },
+  paper: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    height: "100%"
+  },
+  textField: {
+    borderColor: "#ffffff"
+  },
+  topButton: {
+    width: 400,
+    height: 45,
+    marginTop: theme.spacing(4),
+    marginBottom: theme.spacing(1)
+  },
+  bottomButton: {
+    width: 400,
+    height: 45,
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(1)
+  },
+  logo: {
+    marginTop: theme.spacing(8),
+    marginBottom: theme.spacing(3)
+  },
+  main: {
+    height: "100%"
+  },
+  whiteText: {
+    color: "white"
+  },
+  demo: {
+    backgroundColor: theme.palette.background.paper
+  },
+  navigator: {
+    color: "#ffffff",
+    marginTop: theme.spacing(4),
+    marginLeft: theme.spacing(4),
+    fontSize: 35
+  }
+}));
+
+const ImportHexKey = () => {
```

Comment:
Why is this here, we only have mnemonic keys now

Reply:
hmm, probably a rebase thing. If it got deleted on dev and I made a change there, during rebase auto-merge would ignore the delete and just apply my change. (I think)